src/brevitas/graph/calibrate.py

```diff
@@ -164,101 +137,352 @@ def apply(self, model):
         return model
 
 
-class DisableEnableQuantization(Transform):
-
-    def __init__(self, call_act_quantizer_impl=False):
-        super(DisableEnableQuantization, self).__init__()
-        self.disable_act_quant_hooks = []
-        self.call_act_quantizer_impl = call_act_quantizer_impl
+class DisableEnableQuantization:
```

Comment:
Let's rename to QuantizationStatusManager or something like that

This would be a breaking change.